components/Navbar.js

```diff
@@ -21,7 +21,7 @@ export default function Navbar() {
       label: "Contact",
     },
   ];
-  const QUICKET_LINK = "https://qkt.io/pyconug2023";
+  const QUICKET_LINK = "#";
```

Comment:
making this a link to the call for proposals

button test as `Become a Speaker`
href="https://sessionize.com/pycon-uganda"

Reply:
@Amanya-Brian @basiima

Reply:
we can revert this to move back to a `get ticket` button when we are ready

Reply:
I agree with the proposed change. It makes for better UX